lib/worker.py

```diff
@@ -287,7 +287,7 @@ def task_get(self, task_queue):
 
     @staticmethod
     def is_joinable(task_queue):
-        return 'task_done' in task_queue.__dict__.keys()
+        return 'task_done' in list(task_queue.__dict__)
```

Comment:
Same as above. Don't transform a dict to something else to check a key presence. Use `'task_done' in task_queue.__dict__`. Or, better, leave `.keys()` when it is already there: it works on Python 2 and Python 3 both.

Reply:
reverted change